viewer/packages/api/src/public/migration/renderOptionsHelpers.ts

```diff
@@ -90,6 +90,32 @@ export function determineSsaoRenderParameters(
   return ssaoParameters;
 }
 
+/**
+ * Determines the default resolution threshold based on device .
+ * @param rendererResolutionThreshold User provided resolution cap.
+ * @param device Device type (mobile, tablet, desktop, etc.).
+ * @param devicePixelRatio Device pixel ratio set on the renderer.
+ */
+export function determineResolutionCap(
+  rendererResolutionThreshold: number | undefined,
+  device: DeviceDescriptor,
+  devicePixelRatio: number
+): number {
+  if (rendererResolutionThreshold) {
+    return rendererResolutionThreshold;
+  }
+
+  let resolutionThreshold = 1.4e6;
+
+  // This is an attempt at increasing performance for high-resolution mobile devices
+  // by mapping resolution to physical pixel size
+  if (device.deviceType === 'mobile' || device.deviceType === 'tablet') {
+    resolutionThreshold = Math.floor(resolutionThreshold / devicePixelRatio);
```

Comment:
Can we suggest return the value here like `return Math.floor(resolutionThershold / devicePixelRatio);`

Reply:
Yes we can, and yes I should.